src/schemas/40-10007/schema.js

```diff
@@ -234,6 +235,10 @@ const schema = {
               type: 'string',
               enum: ['Female', 'Male'],
             },
+            ethnicity: {
+              type: 'string',
+              enum: ['American Indian or Alaska Native', 'Black or African American', 'Hispanic or Latino', 'Native Hawaiian or Pacific Islander', 'White']
```

Comment:
`enums` should be camelCased. If you want, create a `enumNames` property that hold what you want the user to see.

Reply:
Oh yeah good catch, and then use labels to make them look like this